Auth0 Migration instructions (DO NOT MERGE UNTIL AUTH0 MIGRATION IS LIVE) (#3158)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md

This document details the steps to integrate dbt Cloud with an identity
provider in order to configure Single Sign On and [role-based access control](/docs/cloud/manage-access/about-user-access#role-based-access-control).

> 

Author: ```suggestion
```

This document details the steps to integrate dbt Cloud with an identity
provider in order to configure Single Sign On and [role-based access control](/docs/cloud/manage-access/about-user-access#role-based-access-control).

> <Snippet src="auth0-uri" />

Author: ```suggestion
```

When prompted for the SAML 2.0 application configurations, supply the following values:

> - Single sign on URL: `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`

Author: ```suggestion
* Single sign on URL: `https://YOUR_ACCESS_URL/complete/saml`
```

> - Audience URI (SP Entity ID): `urn:auth0:us-devspace-cluster:{slug}`

Author: ```suggestion
* Audience URI (SP Entity ID): `https://YOUR_ACCESS_URL/`
```

1. On the **SAML Settings** page, enter the following values:

>    - Single sign on URL: `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`

Author: ```suggestion
   * **Single sign on URL**: `https://YOUR_ACCESS_URL/complete/okta`
```

>    - Audience URI (SP Entity ID): `urn:<YOUR_AUTH0_ENTITYID>:{slug}`

Author: ```suggestion
      * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`

>    - Relay State: `<login slug>`

Author: ```suggestion
   * **Relay State**: `<login slug>`
```

>    - Relay State: `<login slug>`

Author: ```suggestion
   * **Relay State**: `<login slug>`
```

1. On the **SAML Settings** page, enter the following values:

   * **Single sign on URL**: `https://YOUR_ACCESS_URL/complete/okta`

>       * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`

Author: ```suggestion
   * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`
```

2. Download the **IDP metadata**.
3. Copy the **SSO URL** and **Entity ID** and download the **Certificate** (or **SHA-256 fingerprint**, if needed).
4. Enter the following values on the **Service Provider Details** window:

>    - Single sign on URL: `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`

Author: ```suggestion
   * **ACS URL**: `https://YOUR_ACCESS_URL/complete/saml`
```

>    - Audience URI (SP Entity ID): `urn:<YOUR_AUTH0_ENTITYID>:{slug}`

Author: ```suggestion
   * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`
```

>    - Relay State: `<login slug>`

Author: ```suggestion
   - **Start URL**: `<login slug>`
```

   | Field | Value |
   | ----- | ----- |
   | **Identifier (Entity ID)** | Base URL for dbt Cloud. Use `https://YOUR_ACCESS_URL/`. |

>    | **Reply URL (Assertion Consumer Service URL)** | Use `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`. |

Author: ```suggestion
   | **Reply URL (Assertion Consumer Service URL)** | Use `https://YOUR_ACCESS_URL/complete/saml`. |
```

3. Under the **Configuration tab**, input the following values:

   - **RelayState:** `<login slug>`

>    - **Audience (EntityID):** `urn:<YOUR_AUTH0_ENTITYID>:{slug}`

Author: ```suggestion
   - **Audience (EntityID):** https://YOUR_ACCESS_URL/
```

>    - **ACS (Consumer) URL Validator:** https://<YOUR_AUTH0_URI>/login/callback?connection={slug}

Author: ```suggestion
   - **ACS (Consumer) URL Validator:** https://YOUR_ACCESS_URL/complete/saml
```

>    - **ACS (Consumer) URL:** https://<YOUR_AUTH0_URI>/login/callback?connection={slug}

Author: ```suggestion
   - **ACS (Consumer) URL:** https://YOUR_ACCESS_URL/complete/saml
```
